pkg/sysregistriesv2/shortnames.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/BurntSushi/toml"
 	"github.com/containers/image/v5/docker/reference"
 	"github.com/containers/image/v5/types"
+	"github.com/containers/storage/pkg/homedir"
```

Comment:
Do we have this anywhere else that needs to be cleansed?

Reply:
Not in here.

Reply:
Once this is cleaned up then we need to vendor new versions into buildah as well.